Procfile.dev

```diff
@@ -1,3 +1,4 @@
 web: PORT=3000 bundle exec puma -C config/puma.rb
+sidekiq: bundle exec sidekiq -q default -q push -q pull -q mailers
```

Comment:
will this need a corresponding announcement and docs update? Can we have an obvious admin error message if people aren't running the all of the queues?